src/bitcoind.cpp

```diff
@@ -153,17 +172,6 @@ static bool AppInit(NodeContext& node, int argc, char* argv[])
     std::any context{&node};
     try
     {
-        if (auto error = common::InitConfig(args)) {
```

Comment:
In commit "refactor: decouple early return commands from AppInit" (61fe1af01418d8b93d38aba8aeb65aecfa6a95cd)

Note for reviewers: Moving the `common::InitConfig` call out of the try block means if InitConfig throws an exception it won't be caught. But this is good and should not change behavior because InitConfig is already catching `std::exception` internally and should just be doing basic parsing and not triggering other types of exceptions.

Reply:
yeah, same note applies for the other loop that was moved out of the try catch. The one that calls `IsSwitchChar()`.